utilities/backupable/backupable_db.cc

```diff
@@ -1407,16 +1407,19 @@ Status BackupEngineImpl::InsertPathnameToSizeBytes(
     std::unordered_map<std::string, uint64_t>* result) {
   assert(result != nullptr);
   std::vector<Env::FileAttributes> files_attrs;
-  Status status = env->GetChildrenFileAttributes(dir, &files_attrs);
-  if (!status.ok()) {
-    return status;
+  Status status = env->FileExists(dir);
+  if (status.ok()) {
+    status = env->GetChildrenFileAttributes(dir, &files_attrs);
```

Comment:
The HDFS client logs an error saying the file doesn't exist. Two customers got confused by these extraneous errors in their logs over the 1-2 weeks, so I'm trying to get rid of them.

At least for this patch, it won't add significant number of calls since this function is only used a few places when initializing engine or creating a new backup.